source/transfer_list.rst

```diff
@@ -657,5 +658,84 @@ such that the last ACPI table in this entry ends at offset
      - hdr_size
      - One or more ACPI tables.
 
+.. _arm_entries:
+
+Arm specific entries
+^^^^^^^^^^^^^^^^^^^^
+
+The following entry types are defined for Arm projects,
+including TF-A, OP-TEE and FF-A:
+
+**OP-TEE pagable part entry layout (XFERLIST_OPTEE_PAGABLE_PART)**
```

Comment:
I guess to make it more precise, we can use the term 'the base address of OP-TEE pagable part', however, it is a bit difficult to give more information about it since it should be described in the design document of OP-TEE and out of scope in this document. Maybe a link to the OP-TEE design page is a good idea.

Reply:
A link would help, but I think this should explain here what a pagable part is. That is the bit I don't get. It sounds like it means something, but what? How does one choose the value to put here? Why does it have just an address, and not a size as well?

Reply:
Following the way we were introducing FDT and HOB, I added `[OPTEECore]` and `[ArmFFA]` links for reference.
I keep the term `OP-TEE pageable part address` to align with the name that is usually called and explain it as `the address of OP-TEE pageable part` in the descriptions and tables.
I think this is fair enough and not necessary to explain in this spec why only an address is needed or why the pointed memory is not required  - it will be too verbose and out-of-scope and they should be described in the related OP-TEE docs (Similar with what we did for FDT and HOB, we didn't even explain what they are in this spec).